Libs/RaptureCommon/src/main/java/rapture/common/impl/jackson/JacksonUtil.java

```diff
@@ -139,6 +162,13 @@ public static String jsonFromObject(Object object) {
         return object;
     }
 
+    /**
+     * Very inefficient.
```

Comment:
It's a comment for anyone who wants to know why the method is deprecated. 
AFAICT it's only ever used in conjunction with jsonFromObject so I added a flag and a better implementation and changed some of the calls (only a few unit tests still use it)